### ISSUE TITLE
[Feature] systemd is sus, eject it

### ISSUE THREAD

**Issue by RockfordRoe** [Amog-OS/AmogOS#139]
**Is your feature request related to a problem? Please describe.**
i saw systemd vent to the nsa

**Describe the solution you'd like**
eject systemd 

**Describe alternatives you've considered**
use susvinit/openrc/runit/sheppard instead

**Additional context**
redhat is sussy

**Comment by IAMACAR10** [comment]
if amogOS was based on devuan systemd (the imposter) could be ejected

**Comment by buddyjojo** [comment]
enhancement more like cripple

**Comment by buddyjojo** [comment]
xd

**Comment by IAMACAR10** [comment]
being serious for a second, systemd isnt bad. like i said in the discord (pisscord), extermist linux users feel the need to ovveract to tiny things like the init system. who cares if your computer uses 20mb less ram or starts up 1 second faster? it really doesn't matter so much

**Comment by RockfordRoe** [comment]
I know this whole thing's a shitpost, but I just wanted to throw in a few serious comments

> if amogOS was based on devuan systemd (the imposter) could be ejected

My gripe with Devuan is that "freedom of init choice" excludes systemd and purges it from its repos IIRC, Being a Gentoo user, I much prefer its approach of allowing the user to decide the init system themselves. I could personally just make a Gentoo-based AmogOS ISO, but that'd be kinda overkill. 

> extermist linux users feel the need to ovveract to tiny things like the init system. who cares if your computer uses 20mb less ram or starts up 1 second faster? it really doesn't matter so much

[Dats da jök](https://youtu.be/xECUrlnXCqk)

**Comment by IAMACAR10** [comment]
Yeah, i agree. If "freedom" means excluding a choice then it isn't really a choice. If I let you choose a drink, but excluded water, I am not really letting you make a choice. Same with the joke, even as I sent it i kinda felt I was overreacting - sorry

**Comment by justleoo** [comment]
ya dont use systemd, is so sus
